amr-wind/wind_energy/ABLStats.H

```diff
@@ -137,6 +133,11 @@ private:
     //! Number of cells in the horizontal direction
     size_t m_ncells_h1{0};
     size_t m_ncells_h2{0};
+
+#ifdef AMR_WIND_USE_NETCDF
+    //! Calculate and write sfs stress averages to output
+    void output_sfs_stress_avgs(ncutils::NCGroup grp, const size_t nt);
+#endif
```

Comment:
So looking at the logic in the code it seems like we can hide this from `ABLStats.H` and just make it a standalone function inside ABLStats.cpp

Reply:
Addressed in https://github.com/Exawind/amr-wind/pull/290/commits/da05df00d9686f43b32742e3ec13693811db1ab3